examples/multisig/virtual.eg.ts

```diff
@@ -31,13 +32,28 @@ const { alexa, billy, carol, david } = await createDevUsers()
 /// With this state, we can hydrate from / use an existing virtual multisig.
 /// Let's see if we can hydrate the virtual multisig state from command line arguments.
 /// If we haven't specified any virtual multisig state, we deploy a new virtual multisig.
-let { state } = parse(Deno.args, { string: ["state"] })
+const args = parse(Deno.args, { string: ["state"], boolean: ["use-stash"] })
+let state = args.state
+const useStash = args["use-stash"]
+
 if (!state) {
+  const stashAccountId = useStash
+    ? await polkadotDev.Proxy
+      .createPure({ proxyType: "Any", delay: 0, index: 0 })
```

Comment:
Iirc, `delay` is optional / can be omitted?

Reply:
Nope its required

```ts
Argument of type '{ proxyType: "Any"; index: number; }' is not assignable to parameter of type '{ proxyType: ProxyType | Rune<ProxyType, never>; delay: number | Rune<number, never>; index: number | Rune<number, never>; }'.
  Property 'delay' is missing in type '{ proxyType: "Any"; index: number; }' but required in type '{ proxyType: ProxyType | Rune<ProxyType, never>; delay: number | Rune<number, never>; index: number | Rune<number, never>; }'.
```